### DISCUSSION TITLE
Log file location

### DISCUSSION THREAD

**Discussion by krisrok** [awawa-dev/HyperHDR#725] [General]
Hi, is the log written to a file somewhere? I have issues during process start-up I wanted to investigate but I cannot find the logs. Starting with `-c` does not help either as the window vanishes rightaway.

**Comment by awawa-dev** [comment]
Hi, for which system? journalctl -r?

**Reply by krisrok** [reply]
Ah, sorry, Windows.

**Reply by awawa-dev** [reply]
--console doesn work? it rather may only happen for custom builds after some changes have been applied to code.

**Reply by krisrok** [reply]
--console spawns a new console that emits some logs but then terminates on its own.

**Reply by krisrok** [reply]
Usually when I try to start it again, it works after a few retries.

**Reply by awawa-dev** [reply]
Maybe there is already a process named like HyperHDR in the background and that's why it quits immediately, check Windows the process manager.
